stack/maximaparser/autogen/parser.mbstring.php

```diff
@@ -2266,7 +2266,7 @@ private function peg_parseAnnotationDetail() {
         }
       }
 
-      $this->peg_cache[$key] = array ("nextPos" => $this->peg_currPos, "result" => $s0 );
+      $this->peg_cache[$key] =  ["nextPos" => $this->peg_currPos, "result" => $s0];
```

Comment:
array () is adding an extra space. Need to check autogen code again. Also why is this not flagged by code check?

Reply:
@smmercuri Looks like we should not be changing the folders listed in thirdpartylibs.xml. They're not ours and code check ignores them. No need to check them. I'll revert them once we've done our first pass.